xbmc/cores/dvdplayer/DVDCodecs/Overlay/contrib/cc_decoder708.cpp

```diff
@@ -290,6 +290,7 @@ void updateScreen (cc708_service_decoder *decoder)
       left=0;
     int copyrows=top + wnd[i]->row_count >= I708_SCREENGRID_ROWS ?
         I708_SCREENGRID_ROWS - top : wnd[i]->row_count;
+    copyrows = copyrows > I708_MAX_ROWS ? I708_MAX_ROWS : copyrows;
```

Comment:
Might as well make that "if copyrows > I708_MAX_ROWS copyrows = I708_MAX_ROWS " , or use std::min